vendor/modules.txt

```diff
@@ -168,6 +168,7 @@ github.com/envoyproxy/go-control-plane/envoy/service/discovery/v2
 github.com/envoyproxy/go-control-plane/envoy/config/grpc_credential/v2alpha
 github.com/envoyproxy/go-control-plane/envoy/config/bootstrap/v2
 github.com/envoyproxy/go-control-plane/envoy/service/tap/v2alpha
+github.com/envoyproxy/go-control-plane/envoy/config/rbac/v2
```

Comment:
should the v2alpha one be removed? We don't have a linter for this yet but you may need to run `go mod tidy` -- or maybe not, I don't know how this is generated

Reply:
I did run `mod tidy`